datafn/python/tests/test_mutation_enhanced.py

```
@@ -0,0 +1,446 @@
+                {"id": "t1", "title": "A"},
+                {"id": "t2", "title": "B"},
+            ],
+        })
+        assert res["ok"] is True
+        assert res["result"]["affectedIds"] == ["t1", "t2"]
+        tasks = await db.find_many("tasks", [])
+        assert len(tasks) == 2
+
+    @pytest.mark.asyncio
+    async def test_batch_insert_respects_batch_size_limit(self):
+        handler, db = await _setup(limits={"maxBatchSize": 1})
```

Ruff (0.15.7)

[warning] Unpacked variable `db` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)